engine/apps/alerts/escalation_snapshot/utils.py

```diff
@@ -1,29 +1,37 @@
+import datetime
+import typing
+
 import pytz
 from django.utils import timezone
 
 
-def eta_for_escalation_step_notify_if_time(from_time, to_time, current_time=None) -> timezone.datetime:
```

Comment:
this function is called once, and `current_time` is never passed in, getting rid of it